chain-impl-mockchain/src/testing/e2e/evm_mapping.rs

```diff
@@ -0,0 +1,56 @@
+use crate::testing::scenario::wallet;
+use crate::testing::TestGen;
+use crate::testing::{scenario::prepare_scenario, verifiers::LedgerStateVerifier, ConfigBuilder};
+use chain_evm::Config;
+
+const ALICE: &str = "Alice";
+const BOB: &str = "Bob";
+
+#[test]
+pub fn evm_mapping() {
```

Comment:
this is very good test but i think it's testing two things:

- user can send correct evm mapping tx sending
- user cannot override evm mapping once its set

IMHO we should have 2 test cases